### ISSUE TITLE
No Penryn?

### ISSUE THREAD

**Issue by tertaco** [Pavo-IM/OC-Gen-X#120]
Dortania has guide for Penryn CPU, but there's no option for them here (you really should add them)

**Comment by osx86-ijb** [comment]
> Dortania has guide for Penryn CPU, but there's no option for them here (you really should add them)

 - First: That’s Dortania, not him. 
 - Second: It’s his software, therefore his choice. 
 - Third: “You really should add them” is not a way to ask someone nicely, is it?
 - Fourth: Why not fork the project and contribute to it thereby adding the support for it yourself, instead of attempting to boss someone else around through opening an issue on GitHub, (whilst both simultaneously and incorrectly using the ability to do so, as this **isn’t technically an issue**?)

**Comment by tertaco** [comment]
1. I wrote it at night and didn't really pay attention to sentence construction.
2. Yeah, but why not consider adding it?
3. English isn't my native language (which means I'm not really good with phrases and stuff), I'm sorry if it sounded bad.
4. I'm not a coder (this is why I suggested adding stuff). Again, I'm trying to be nice as much as I can, but using proper phrases is my nightmare. And again, I was writing it at night, so I missed the right category. Sorry for doing stuff wrong.
